container/id/id.go

```diff
@@ -10,39 +10,41 @@ import (
 )
 
 // ID represents v2-compatible container identifier.
```

Comment:
```suggestion
// ID represents container identifier.
```

Reply:
why? thought all public types should have such info